action/protocol/staking/contractstake_indexer.go

```diff
@@ -86,6 +86,11 @@ func (c *delayTolerantIndexer) wait(height uint64) (bool, error) {
 	if h >= height {
 		return true, nil
 	}
+	if indexWithStart, ok := c.ContractStakingIndexer.(interface{ StartHeight() uint64 }); ok {
+		if indexWithStart.StartHeight() >= height {
+			return false, nil
+		}
+	}
```

Comment:
add a `startHeight` in `delayTolerantIndexer` struct, and move this type assertion (and save its start height) when creating the instance one-time, so can directly use the value